### DISCUSSION TITLE
Script Help

### DISCUSSION THREAD

**Discussion by gingerninja2000** [ImageMagick/ImageMagick#4831] [Consulting]
Looking for help with a very basic script. I have a folder of seamless tile images for use on printed fabrics.

I want to create a preview image of each tile in a 3x3 grid so people can see the repeat of the pattern.

I want to be able to run the script in a folder and all the images in that folder have their own preview 3x3 as a file with the same name but with preview included at the end of the filename.

I can run a command to a single image to make it work but I need help making it run on a folder full of images.

**Comment by snibgo** [comment]
This seems to be the question you have already asked in https://github.com/ImageMagick/ImageMagick/discussions/4827

Please don't multi-post.

**Comment by gingerninja2000** [comment]
Hi

I posted in a different category (paid specialist as I'm willing to pay for
the help)

I cant' get the for command to work. Can you help me with this and I'll pay
for your time.
